lib/fips_lookup.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
```

Comment:
Renamed file due to fork rename, can compare to `lib/fips_code_lookup.rb` as main working class for the gem